docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -88,27 +129,48 @@ wp-env --version
 > 1.6.0
 ```
 
-The `wp-env` script is used to create a Docker WordPress environment, to use start from your plugin directory—if you are following the create block tutorial, this would be in the generated directory.
+The `wp-env` script is used to create a Docker WordPress environment. You can use this script to start an environment from your plugin directory, for example if you are following the create block tutorial, this would be in the generated directory like so:
```

Comment:
```suggestion
The `wp-env` script is used to create a Docker WordPress environment. You can use this script to start an environment with your plugin activated by running it from the directory containing your plugin. For example if you are following the create block tutorial, this would be in the generated directory like so:
```